lib/errors.js

```diff
@@ -130,6 +130,11 @@ const codes = {
     'Cannot specify both logger.stream and logger.file options'
   ),
 
+  FST_ERR_LOG_INVALID_OBJECT: createError(
```

Comment:
I think you dont need this error. You are not throwing at runtime but on startup. Fastify errors are for runtime errors, thats why they have also statusCode 500 by default. Remove this error and use a normal TypeError.

Reply:
Not really, see `FST_ERR_LOG_INVALID_DESTINATION`.
This file is used to store all errors.

Reply:
Then why are not all thrown errors in e.g. fastify.js created by fastify-errors?

Reply:
No reason, maybe `fastify-errors` is not exist on that point or just forgotten we have a `lib/errors` when we implement the feature.

Reply:
I created #4521 to discuss it.